crates/polars-time/src/chunkedarray/rolling_window/mod.rs

```diff
@@ -98,25 +105,7 @@ impl From<RollingOptions> for RollingOptionsImpl<'static> {
             tz: None,
             closed_window: None,
             fn_params: options.fn_params,
-        }
-    }
-}
-
-impl From<RollingOptions> for RollingOptionsFixedWindow {
```

Comment:
this is unused - OK to just remove?

Reply:
Yes, there is a no unused code policy. :police_officer: